src/tests/load-function-module.test.ts

```diff
@@ -70,22 +28,20 @@ Deno.test("LoadFunctionModule function", async (t) => {
     );
   });
 
-  await t.step("should throw if does not exist", async () => {
-    const jsModule = await LoadFunctionModule(
-      [`${functionsDir}/nonexistnent.ts`],
-    );
-    assertEquals(
-      jsModule,
-      null,
-      "Could not load function module for function: nonexistent",
+  await t.step("should throw if file does not exist", async () => {
+    await assertRejects(
+      async () => {
+        return await LoadFunctionModule(`${functionsDir}/nonexistnent.ts`);
+      },
+      "Module not found",
```

Comment:
This is the test covering the main change in this PR: 'Module not found' exceptions bubbling up to the user.